### ISSUE TITLE
SRV Records not working

### ISSUE THREAD

**Issue by kahootali** [zentyal/zentyal#1952]
**Describe the bug**
I am trying to create Openshift Cluster but it needs following SRV records to be created
```
# _service._proto.name.                            TTL    class SRV priority weight port target.
_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-0.<cluster_name>.<base_domain>.
_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-1.<cluster_name>.<base_domain>.
_etcd-server-ssl._tcp.<cluster_name>.<base_domain>  86400 IN    SRV 0        10     2380 etcd-2.<cluster_name>.<base_domain>.
```
but it is not working. 

I have added in Networks -> Services -> Add New -> 
```
Protocol: TCP/UDP           etcd-server-ssl        Source Port: Any            Destination Port: 2380
```

And In Domain -> Services 

![image](https://user-images.githubusercontent.com/13013437/77521897-30773780-6ea5-11ea-9feb-88a3770d1ad9.png)

But it is not working, I am trying

```
$ host -t SRV _etcd-server-ssl._tcp
Host _etcd-server-ssl._tcp not found: 3(NXDOMAIN)
```

or even full url

```
$ host -t SRV _etcd-server-ssl._tcp.<subdomain>.<domain>
Host _etcd-server-ssl._tcp.<subdomain>.<domain> not found: 3(NXDOMAIN)
```

Zentyal: 6.1

**Comment by brunolorente** [comment]
Hi @kahootali ,

We have reproduced the issue, and I have added it to our roadmap. 
On this link, you'll find a workaround to keep managed your domain: https://wiki.samba.org/index.php/DNS_Administration#Adding_new_records

Thank you very much for your contribution!

**Comment by brunolorente** [comment]
Hi @kahootali ,

I created a new lab in order to test the bug reported and I couldn't reproduce it. I tested your steps with Zentyal 3.2, 4.2, 5.0, 6.1 and I always got the right answer. The steps I did were:

1. I configured the DNS and Domain Controllers modules.
2. I added the 'etcd-server-ssl' service to the configuration file '/etc/services'.
3. I set a few SRV records for this service.
4. I checked that I was able to reach the SRV records.

So, I recommend you to check the log files ('/var/log/zentyal/zentyal.log' and '/var/log/syslog').

Finally, if the issue remains, you can open a new thread in the forum or contact with sales@zentyal.com for official supporting.

**On the other hand** thanks to this issue, I found a bug related with the NS records, the behaviour was the same that you described with the SRV records but within NS records, It have been fixed in e1071ef and will be released in a few days, at the same time than Zentyal Server 6.2.

Best regards, and many many thanks for your time!

**Comment by Neustradamus** [comment]
@kahootali: Can you test the fix?
